sledgehammer/start-up.sh

```diff
@@ -149,6 +149,11 @@ echo "# Sledgehammer added to log to the admin node" >> /etc/rsyslog.conf
 echo "*.* @@${ADMIN_IP}" >> /etc/rsyslog.conf
 service $RSYSLOGSERVICE restart
 
+# Sometimes at this point network is not up yet, wait for it
+ping -c 1 -w 60 $ADMIN_IP > /dev/null || {
```

Comment:
I verified this and the way it was written seems to be the correct one. With `ping -c 1 -w 60` we are not sending one packet, we are sending packets for 60 seconds and wait a single reply. If a single reply arrives before the deadline (`-w 60`), ping exits successfully. Otherwise one packet per second will be sent up to 60 and then exit with failure.

I have tested this manually and seems to be working as expected.

from ping man:
```
-w deadline
              Specify a timeout, in seconds, before ping exits regardless of how many packets have been sent or received. In this case ping does not stop after
              count packet are sent, it waits either for deadline expire or until count probes are answered or for some error notification from network.
```